metadata.rb

```diff
@@ -4,7 +4,7 @@
 license          'Apache 2.0'
 description      'Installs/Configures redis'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version          '2.2.4'
+version          '2.2.5'
```

Comment:
please do not include the version number when submitting changes there are several reasons not to:
- the maintainer could disagree with you on impact
- PRs are not a FIFO queue so other PRs merging with unrelated changes now forces a rebase in order to proceed (like in this case)
- some projects lump multiple changes into a single release (not that I am a fan on this)